packages/job-manager/src/worker/worker.spec.ts

```diff
@@ -123,10 +130,24 @@ describe(Worker, () => {
             })
             .verifiable(Times.exactly(taskCount));
 
+        const expectedMeasurements: BatchPoolMeasurements = {
+            runningTasks: poolMetricsInfo.load.runningTasks,
+            samplingIntervalInSeconds,
+            maxParallelTasks: poolMetricsInfo.maxTasksPerPool,
+        };
+
+        loggerMock
+            .setup(lm =>
+                // tslint:disable-next-line: no-null-keyword
+                lm.trackEvent('ScanTasksQueued', null, expectedMeasurements),
```

Comment:
move all the event names to a file. TelemetryEventNames

Reply:
you can also see if you can add strict typings for the event content based on event name.